src/app/address.ts

```diff
@@ -0,0 +1,8 @@
+export interface Address {
```

Comment:
Would suggest putting all interfaces in an interface folder so it won't clutter the app folder